x-pack/plugins/enterprise_search/server/lib/indices/create_api_key.ts

```diff
@@ -17,14 +17,22 @@ export const createApiKey = async (
   indexName: string,
   keyName: string
 ) => {
+  let aclIndexName: string;
```

Comment:
nitpick: more idiomatic for our codebase would be
```
const aclIndexName = indexName.startsWith('search-') ? `.search-acl-filter-${indexName.slice(7)}` : `.search-acl-filter-${indexName}`;
```

Reply:
I can make that change. I hadn't considered using a substring method so I could inline the suffix extraction with the prefix concatenation. 

If I _had_ to have a block of logic, what's the idiomatic way to do this to avoid the `let`? (I assume that's what you're trying to avoid?)

Reply:
An IIFE is possible but isn't really idiomatic, we're more likely to extract it to a function elsewhere in the file and call it here (or to a shared utility function that's used in multiple places, which might be useful here).